recipes/dnp-binstrings/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: binstrings
+  version: "1.0"
+
+source:
+  url: https://github.com/erinijapranckeviciene/dnpatterntools/files/2450741/v1.0.tar.gz
+  sha256: 19c46d6bc6a1af70e5a2b4a051aab334026dc38c84b367ea9c445da3fdb21cfe
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - seqan-library
```

Comment:
Seqan should probably in the host section.

Reply:
Thank you for the comment. It was a journey... No. Seqan is C++ header only library. It matters only in compilation.  But it depends on zlib and bzip2. The two should be in the host and run.